theano/gpuarray/basic_ops.py

```diff
@@ -968,7 +960,7 @@ class GpuAllocEmpty(HideC, AllocEmpty):
     _f16_ok = True
     params_type = gpu_context_type
 
-    def __init__(self, dtype, context_name):
+    def __init__(self, dtype, context_name=None):
```

Comment:
@abergeron I'm not sure of this change. What do you think? For safety, it is better to not include it, but if it is good, we can get it in.